lib/util/exec/docker/index.ts

```diff
@@ -110,12 +110,23 @@ async function getDockerTag(
   return 'latest';
 }
 
-function getContainerName(image: string): string {
-  return image.replace(/\//g, '_');
+export function getContainerName(image: string, prefix: string): string {
+  if (prefix) {
+    return prefix.concat(image.replace(/\//g, '_'));
+  }
+  return 'renovate_'.concat(image.replace(/\//g, '_'));
+}
+
+export function getContainerLabel(prefix: string): string {
+  if (prefix) {
+    return `${prefix}child`;
+  }
+  return 'renovate_child';
```

Comment:
```suggestion
export function getContainerName(image: string, prefix: string): string {
  return `${prefix}${image.replace(/\//g, '_'}`;
}

export function getContainerLabel(prefix: string): string {
    return `${prefix}child`;
```

Maybe we can inline this?